releaf-core/spec/lib/error_formatter_spec.rb

```diff
@@ -46,7 +46,14 @@ def base_validation
       b.valid?
       b
     }
-    let(:blank_error) { {error_code: :blank, full_message: "Blank"} }
+
+    def blank_error(attribute, class_name='DummyResourceValidatorBook', id='null')
```

Comment:
Line is too long. [82/80]<br>Surrounding space missing in default value assignment.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.